O/oneTBB/bundled/patches/i686-musl.patch

```diff
@@ -0,0 +1,10 @@
+--- a/cmake/compilers/GNU.cmake
++++ b/cmake/compilers/GNU.cmake
+@@ -50,7 +50,6 @@
+ if (NOT ${CMAKE_CXX_COMPILER_ID} STREQUAL Intel)
+     # gcc 6.0 and later have -flifetime-dse option that controls elimination of stores done outside the object lifetime
+     set(TBB_DSE_FLAG $<$<NOT:$<VERSION_LESS:${CMAKE_CXX_COMPILER_VERSION},6.0>>:-flifetime-dse=1>)
+-    set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} $<$<NOT:$<VERSION_LESS:${CMAKE_CXX_COMPILER_VERSION},8.0>>:-fstack-clash-protection>)
```

Comment:
If you apply this patch when targeting Windows you can also keep GCC 9, instead of using GCC 13, and sidestep the ICE.

Reply:
Yes – I thought it's nicer to use a GCC that doesn't report an ICE. Do you prefer GCC 9?

Reply:
In general I'd prefer older GCCs, especially since this a C++ project, which is a compatibility nightmare.

Reply:
Ok.